src/misc/m.tnm.download/m.tnm.download.py

```diff
@@ -164,6 +166,11 @@
 )
 
 
+def urlopen(url):
+    url = url.replace(" ", "%20")
+    return urllib.request.urlopen(url)
```

Comment:
There is probably encode-decode function which would do this in more general way.

Reply:
Found `url = urllib.parse.quote(url, safe=":/?=&")`, but I liked the above simpler version.

Reply:
I was actually surprised that I couldn't find any general URL encode/decode functions that can simply take *full* URLs.

Reply:
Oh BTW, TNM only complained about spaces in URL. That was another reason.

Reply:
Then I don't have a better suggestion. I think a documentation would clarify the existence of the function or a separate function `encode_spaces()` or `encode_X_for_Y()` as there seems to be some specificity to this particular case.